t/find_link.t

```diff
@@ -148,14 +148,22 @@ isa_ok( $x, 'WWW::Mechanize::Link' );
 $x = $mech->find_link( url_abs => 'blongo.html' );
 ok( !defined $x, 'No match' );
 
-$x = $mech->find_link( url_abs_regex => qr[t/blongo\.html$] );
+$x = $mech->find_link( url_abs_regex => qr[t/blongo\.html$] ); #]
```

Comment:
Is this change a typo?

Reply:
Ah, I forgot to take that out. I recently switched to VSCode on my $work computer for the live share plugin I need for mentoring, and this regex broke syntax highlighting. I will remove and force-push it.

Reply:
Ah. I understand now. We now have a slightly different issue:

```
#   Failed test 'No incorrect line endings in 't/find_link.t' on line 151: [\s]'
#   at xt/author/eol.t line 108.
#   Problem Lines: 
# line 151: [\s]: $x[\s]=[\s]$mech->find_link([\s]url_abs_regex[\s]=>[\s]qr[t/blongo\.html$][\s]);[\s]
# Looks like you failed 1 test of 96.
xt/author/eol.t ..........................
```

Reply:
Have you tried different regex delimiters or `\z` rather than `$`?

Reply:
As usual, I forgot to run the author tests. There was an extra whitespace at the end. Sorry about that. :facepalm: